pennylane/templates/broadcast.py

```diff
@@ -142,7 +146,8 @@ def broadcast(unitary, wires, pattern, parameters=None, kwargs=None):
         unitary (func): quantum gate or template
         pattern (str): specifies the wire pattern of the broadcast
         parameters (list): sequence of parameters for each gate applied
-        wires (Sequence[int] or int): wire indices that the unitaries act upon
+        wires (Sequence[int] or int): wire indices that the unitaries act upon. Also accepts
+            :class:`pennylane.wires.Wires` objects.
```

Comment:
You could also add to the type hint in the brackets, e.g.,
```rst
wires (Sequence[int] or int or pennylane.wires.Wires)
```
I think sometimes the documentation builds so that it makes a clickable link, but not sure exactly on the syntax.

Reply:
This happens in a lot of cases below :laughing: I'm neutral but I think the brackets could help

Reply:
Hmm, I was unsure about this. Users will never need to feed Wires into a template. The hint is more for developers, so it's a bit hidden...

Reply:
Ok, I just find it not so hidden :laughing:, so I think that the brackets then look incomplete if wires can also be another type beyond `Sequence[int]` or `int`.

And I should really check out the `Wires` class, but is it something that users may eventually use? Although I just looked and `Wires` isn't in the docs, is that on purpose?

Reply:
Convinced, let me add Wires as a possible type there!

Reply:
Actually, I cleaned up this arg description everywhere. Thanks for the suggestion!